targets/XPlatCppSdk/source/code/source/playfab/PlayFabIOSHttpPlugin.mm

```diff
@@ -0,0 +1,346 @@
+#include <stdafx.h>
```

Comment:
Shouldn't this whole file be wrapped in:
#if defined(PLAYFAB_PLATFORM_IOS)
?

Reply:
I followed the implementation of other platforms. But, yes it will be better to be wrapped or make subdirectories. It will be good to do once later.